stats.py

```diff
@@ -297,6 +299,74 @@ def basename():
   print('wrote:', stat)
   return stat
 
+def print_candidate_stats(hitcounts, info):
```

Comment:
I would rename it to inst_fusions()
I wouldn't stress 'print' and stats is module's name.